pkg/stanza/operator/input/windows/operator.go

```diff
@@ -193,6 +196,21 @@ func (e *Input) read(ctx context.Context) int {
 
 // processEvent will process and send an event retrieved from windows event log.
 func (e *Input) processEvent(ctx context.Context, event Event) {
+	if len(e.excludeProviders) > 0 {
```

Comment:
User can use both `exclude_providers` and `raw`. You need to check first if the event should be excluded from processing or not. You don't want to give a raw data of event that the user want to exclude from processing. The return is when the event has an excluded provider - in that case you need to stop processing even if the user chose raw.

Reply:
Ok, understood. We need to unmarshal the event in order to determine if it is excluded or not. 

That makes sense but I see some performance implications here:
1. One of the primary reasons a user may be using `raw` is because they find unmarshaling to be unacceptably costly. So it would seem the user must make a decision based on the trade off here. Either send all events at minimal cost per event, or unmarshal all events but only _send_ those that are not excluded. I think either choice could be reasonable but users should be informed of this tradeoff in the readme so they can make an informed decision. WDYT?
2. If not using `raw`, we are unmarshaling all events (include or excluded) once, and then again unmarshaling those that were not excluded. At a minimum, we should keep a handle to the `simpleEvent` so that we don't have to perform a redundant unmarshal.

Reply:
I agree. I can add to the exclude_providers field:
`One or more event log providers to exclude from processing. Each event log will be processed in order to determine its provider name.`